### DISCUSSION TITLE
Custom email

### DISCUSSION THREAD

**Discussion by RDIL** [community/community#21848] [Other Feature Feedback, Questions, & Ideas]
<p>Hello all,<br>
I was wondering if it is possible to get custom emails for my GitHub pages site (for example: hi@my.domain rather then <a href="mailto:helloworld@gmail.com">helloworld@gmail.com</a>). If you know how to do so, please let me know.<br>
Thanks! :slight_smile:</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>Yes, it is possible but not through GitHub. You would have to contact a service provider for email like Google Apps or FastMail. Whichever service you choose would be able to help you through the setup process.</p>
<p>I hope that helps!</p>

**Comment by DoobleD** [comment]
<p>There is a choice of solutions for that, paid and free, and indeed all of these happen outside of github. On the free side here are a few free services:</p>
<ul>
<li><a href="https://postale.io" rel="nofollow noopener">postale.io</a></li>
<li>
<a href="https://www.zoho.com/mail/zohomail-pricing.html" rel="nofollow noopener">Zoho Mail</a> free plan</li>
<li><a href="https://mail.yandex.com/" rel="nofollow noopener">Yandex</a></li>
</ul>
<p>Zoho Mail free plan limits you to their webmail only, and Yandex has ads. <a href="http://postale.io" rel="nofollow noopener">postale.io</a> doesn’t have those limitations. The DNS setup is also taken care of for you, if requested (still for free).</p>
<p>Hope this helps!</p>
